### DISCUSSION TITLE
Data quality tests

### DISCUSSION THREAD

**Discussion by jddingemanse** [jddingemanse/awtiCodeDev#15] [Ideas]
Let us discuss here about all different data quality tests we know and would like to include. Any relevant ones can be turned into issues.

If possible, do not only mention the name of a specific test, but also include a reference and a clear description / equations of the test.

**Comment by Ayalkie12** [comment]
Double mass curve is a simple, visual and practical method, and it is widely used in the study of the consistency and long-term trend test of hydrometeorological data.
